core/src/main/scala/org/apache/spark/util/ThreadUtils.scala

```diff
@@ -254,4 +258,58 @@ private[spark] object ThreadUtils {
       executor.shutdownNow()
     }
   }
+
+  /**
+   * Transforms input collection by applying the given function to each element in parallel fashion.
```

Comment:
I'd still include a note in these docs about what this does differently from `.par`. Just a sentence about it being interruptible.

Reply:
I added a comment about this.